harmony/__init__.py

```diff
@@ -114,9 +119,12 @@ def _set_current_activity(self, activity=None):
             item(activity_name)
 
     def _send_activity(self, activity):
+        activity = int(activity)
         label = "unknown"
-        if activity in self._activities:
-            label = self._activities[activity]
+        for activity_id, activity_name in self._activities.items():
+            if activity_id == activity:
+                label = activity_name
+                break
```

Comment:
What is the different between the new and the old implementation? At least the add `int` conversion makes sense since the keys are [added as int values](https://github.com/smarthomeNG/plugins/pull/119/files#diff-90b7a0a8b77e4bc5309c66a23782fe53R97).

But shouldn't it be enough to just convert the value to `int` and keep the old implementation in this method?

Reply:
You're right. This is just a relict of a larger code block while debugging. I've changed this to the previous version (+ integer check).